src/utils/tel.ts

```diff
@@ -0,0 +1,24 @@
+function setCursorPosition(pos:any, elem:any) {
+    if (elem.setSelectionRange) elem.setSelectionRange(pos, pos);
+    else if (elem.createTextRange) {
+        let range = elem.createTextRange();
+        range.collapse(true);
+        range.moveEnd("character", pos);
+        range.moveStart("character", pos);
+        range.select()
+    }
+}
+  
+export function mask(event:any) {
+    let matrix = "+7 (___) ___ ____",
```

Comment:
we are looking from different countries

not good idea to make the mask only for russian

Reply:
could be better to skip this validation and just to validate on base rules

We can check:
1. length of number
2. `+` at the start

Reply:
The length of number is different in different countries. Could we check + at the start and that phone length between 10 and 15?